disco_aws_automation/disco_alarm.py

```diff
@@ -10,6 +10,7 @@
 
 # Max batch size for alarm deletion http://goo.gl/vMQOrX
 DELETE_BATCH_SIZE = 100
+ENVIRONMENT_DELETE_SKIP_NAMESPACES = ['AWS/ES']
```

Comment:
If/when we get to implement that feature, this PR would indeed become obsolete. It would not be hard to roll this change back. I'll write down this as AC for such a story.